game/game.go

```diff
@@ -0,0 +1,39 @@
+package game
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+type Guessing struct {
+	guess int 
+	trialcount int 
+}
+
+func (g Guessing) Play(){
+	g.trialcount = 1
+	source := rand.NewSource(time.Now().UnixNano())
+	randomizer := rand.New(source)
+	secrectNumber := randomizer.Intn(10)
+	fmt.Println("Guess the number")
+	for {
+		fmt.Println("Please input your guess")
+		fmt.Scan(&g.guess)
+		if g.trialcount > 3 {
```

Comment:
Make the Constant 3 an instance variable called AllowedCount

Reply:
Added  a new variable named "NumberOfTriesAllowed" that allows dev to adjust attempt count from main.go.

Reply:
Can we call it `AttemptsAllowed`

Reply:
Changes have been made the variable's now called AttemptsAllowed